README.md

```diff
@@ -252,7 +252,7 @@ the JavaScript side, which defaults to 10000 milliseconds (10 seconds). This can
 
 #### Callbacks
 
-When you call an exposed function, you can immediately pass a callback function afterwards. This callback will automatically be called asynchrounously with the return value when the function has finished executing on the other side.
+When you call an exposed function, you can immediately pass a callback function afterwards. This callback will automatically be called asynchronous with the return value when the function has finished executing on the other side.
```

Comment:
Thanks for noticing this spelling mistake. But I think this should be `asynchronously`, right?

Reply:
```suggestion
When you call an exposed function, you can immediately pass a callback function afterwards. This callback will automatically be called asynchronously with the return value when the function has finished executing on the other side.
```